app/helpers/calculators_helper.rb

```diff
@@ -0,0 +1,7 @@
+module CalculatorsHelper
+  def leave_pot_untouched_slider_max(contribution)
+    max = contribution * 5
+    max = 500 if max.zero?
+    max
```

Comment:
Personally feel this would result in a longer less readable line but open to opinions, @benlovell any thoughts?